nubis/puppet/boomi.pp

```diff
@@ -7,6 +7,20 @@
   command => "nubis-cron ${project_name}-boomi-centerstone /usr/local/virtualenvs/boomi/bin/python -m mozilla_etl.boomi.centerstone",
 }
 
+cron::daily { "${project_name}-boomi-CCURE-Redshift":
+  hour    => '18',
+  minute  => '45',
```

Comment:
Well, :45 is that the time you relly want to ?

Why not use the fqdn() approach.. ?